concourse/pipeline.yml

```diff
@@ -39,6 +39,8 @@ jobs:
         params:
           CF_ORG: govuk-accounts
           CF_APP_NAME: govuk-account-static-errors
+          HOSTNAME: govuk-account-static-errors-staging
+          VARFILE_PATH: vars-staging.yml
```

Comment:
I don't think the `VARFILE_PATH` is needed any more

Reply:
urgh, good spot, sorry was messy winding and un-winding this. Missed that bit!

Reply:
@brucebolt resolved: https://github.com/alphagov/govuk-account-static-errors/pull/5/commits/da0e30531ebffa4cf69a41cef0bfcdd853231b6e